src/hybrid_failure_discovery/envs/hovercraft_env.py

```diff
@@ -42,26 +42,38 @@ class HoverCraftSceneSpec:
     dt: float = 0.1
 
     # Initial state hyperparameters.
+    # ORIGINALS
```

Comment:
let's remove these comments -- future users / readers of the codebase won't need to know what was original versus changed in this PR (and that's a good way to determine whether a comment is good to keep)!